aspnetcore/performance/caching/response.md

```diff
@@ -108,7 +108,15 @@ The `ResponseCacheAttribute` is used to configure and create (via `IFilterFactor
 
 This header is only written when the `VaryByHeader` property is set. It's set to the `Vary` property's value. The following sample uses the `VaryByHeader` property:
 
-[!code-csharp[](response/sample/Controllers/HomeController.cs?name=snippet_VaryByHeader&highlight=1)]
+# [ASP.NET Core 2.x](#tab/aspnetcore2x/)
```

Comment:
Please use the versioning monikers instead of tabs. It's confusing to readers if we mix both approaches.

Reply:
This is a major shift from the way that things have been done (tabs have always been used with code blocks). I take it there was an internal discussion and decision on this ... this is the first I'm hearing about it.

Reply:
The agreement was to slowly eliminate tabs once conceptual versioning was implemented. The ASP.NET version tabs were always considered a stopgap solution.